.gitignore

```diff
@@ -6,3 +6,4 @@ doc/
 rdoc/
 tmp
 Gemfile.lock
+.idea
```

Comment:
I think this should be better handled by a `~/.gitignore` file (otherwise we'll snowball any .swc etc stuff here)?